src/org/zaproxy/zap/extension/ascanrules/FormatString.java

```diff
@@ -251,7 +258,7 @@ public void scan(HttpMessage msg, String param, String value) {
 
 
 			
-		} catch (Exception e) {
+		} catch (IOException e) {

```

Comment:
This change should be mentioned in the commit message (like it was done in the PR description).